api/segments/models.py

```diff
@@ -246,6 +247,10 @@ def deep_clone(self, cloned_segment: Segment) -> "SegmentRule":
         cloned_rule.uuid = uuid.uuid4()
         cloned_rule.id = None
         cloned_rule.save()
+        if is_saas():
```

Comment:
1. I sorta agree with you but this was the feedback from @rolodato so I deferred to his judgement.
2. We don't log info in production? That's really surprising. How should I diagnose issues like this where we want to track down problems?

Reply:
Ok I've reverted the commit that adds the `is_saas` check, though I'm not really sure if this PR is useful if we're not logging info in production so I still need guidance on that one.

Reply:
Two solutions come to mind:
1. Enable logging info in production. I don't really see the downside of logging info. It's the log level that's been set at every other company I've been at and I can't be the only engineer that's surprised to see that it's not logged. Disabling debug level logging makes sense though.
2. Switch the logging level to warning which sorta rubs me the wrong way since these aren't really warnings, but I can add comments next to each logging warning call that says basically "these aren't really warnings but we want to log them in production and production is currently set at the warning level" but something more pithier.

Reply:
Yep, I'm happy with (1) - can you add that to this PR? It should just be a case of setting that environment variable in the task definition JSON file (+ the one for staging too for consistency).

Reply:
Ok I think I've set the environment variables for staging and production properly, but I'm not really sure how to test the setting. Maybe we can verify once this is merged into `main` on staging?

Reply:
Yes, exactly.